akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -139,6 +141,30 @@ static void ignoreExample() {
     // #ignore
   }
 
+  static void asPublisherExample() {
+    // #asPublisher
+    Source<Integer, NotUsed> source = Source.range(1, 5);
+
+    Publisher<Integer> publisherTrue = source.runWith(Sink.asPublisher(AsPublisher.WITH_FANOUT), system);
```

Comment:
I don't think it is very valuable to show `WITH_FANOUT`. Only showing creating a stream, and then consuming the stream with fromPublisher should be enough (quite synthetic scenario anyway).